docs/src/components/AppCacheBanner/index.css

```diff
@@ -1,6 +1,6 @@
 .banner {
   position: fixed;
-  position: sticky;
+  position: sticky; /* stylelint-disable-line */
```

Comment:
It's a duplicate declaration, Stylelint would prefer that either `fixed` or `sticky` was specified, not both.